Ugly patch. Avoid strange substitution in CFLAGS/CXXFLAGS. It looks
like shells different than bash fail on those two lines.

diff -burN aMule-2.0.0rc6.orig/configure.in aMule-2.0.0rc6/configure.in
--- aMule-2.0.0rc6.orig/configure.in	2004-10-06 22:52:06.518677128 +0200
+++ aMule-2.0.0rc6/configure.in	2004-10-06 22:52:18.590841880 +0200
@@ -8,8 +8,8 @@
 AM_MAINTAINER_MODE
 
 # No -O2 by default, removing -fomit since it causes problems
-CFLAGS=" ${CFLAGS/-fomit-frame-pointer/}"
-CXXFLAGS=" ${CXXFLAGS/-fomit-frame-pointer/}"
+#CFLAGS=" ${CFLAGS/-fomit-frame-pointer/}"
+#CXXFLAGS=" ${CXXFLAGS/-fomit-frame-pointer/}"
 
 # System detection, need AC_CANONICAL_TARGET
 case "${target_os}" in
